example/src/pages/GlobalUIComp.tsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import { GlobalUI } from "scorer-ui-kit";
+
+const GlobalUIComp = () => {
+  return (
```

Comment:
This should be functional component (use FC)
Name should be GlobalUIComp --> GlobalUIExample

Reply:
Done